src/lib/changelog/data.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * This module deals with building changelogs from serieses.
+ * This module deals with building changelogs from series.
```

Comment:
`series` is its own plural.

You can say `a series of series` if you mean that, or `multiple series` if you think `series` is ambiguous.

Reply:
> You can say a series of series if you mean that, or multiple series if you think series is ambiguous.

Yeah that would be great